test_scripts/WebEngine/commonWebEngine.lua

```diff
@@ -96,6 +105,82 @@ function common.validation(actualData, expectedData, pMessage)
   return true
 end
 
+function common.updatePreloadedPT(pAppId, pAppHMIType)
+  local preloadedTable = common.getPreloadedPT()
+  local appId = config["application" .. pAppId].registerAppInterfaceParams.fullAppID
+  local appPermissions = common.cloneTable(preloadedTable.policy_table.app_policies.default)
+  local WidgetSupport = {
+    rpcs = {
+      CreateWindow = {
+        hmi_levels = { "NONE", "BACKGROUND", "LIMITED", "FULL" }
+      },
+      DeleteWindow = {
+        hmi_levels = { "NONE", "BACKGROUND", "LIMITED", "FULL" }
+      }
+    }
+  }
+  appPermissions.AppHMIType = pAppHMIType
+  appPermissions.enabled = true
+  appPermissions.transportType = "WS"
+  appPermissions.hybridAppPreference = "CLOUD"
+  preloadedTable.policy_table.app_policies[appId] = appPermissions
+  preloadedTable.policy_table.app_policies[appId].groups = { "Base-4", "WidgetSupport" }
+  preloadedTable.policy_table.functional_groupings["DataConsent-2"].rpcs = common.null
+  preloadedTable.policy_table.functional_groupings["WidgetSupport"] = WidgetSupport
+  common.setPreloadedPT(preloadedTable)
+end
+
+function common.disallowedRegisterApp(pAppSessionId)
```

Comment:
expectRegistrationDissallowed

Reply:
@kshala-ford Done in [fd9ef16](https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2432/commits/fd9ef1611657d46d2163603b1abc2770bfb5968b)